README.md

```diff
@@ -1,5 +1,79 @@
 # mhc-fe
-my health connect frontend repository with design studio and frontend source code
+## My health connect frontend repository with design studio and frontend source code
 
+Design tokens are the visual design atoms of the design system — specifically, they are named entities that store visual design attributes, we can say they are variables that can be given a value.
```

Comment:
I still don't see the changes. here is what i am asking. You are starting off with a tutorial. it should have a repository.
this repository is not just for figma tokens. its an entire source code repository. you're part of the documentation should have a heading.

```suggestion
## How to integrate figma, figma token to this repository

Design tokens are the visual design atoms of the design system — specifically, they are named entities that store visual design attributes, we can say they are variables that can be given a value.
```